packages/components/src/RichTooltip/RichTooltip.component.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Actions } from '../Actions';
+import CircularProgress from '../CircularProgress';
+import Icon from '../Icon';
+import Inject from '../Inject';
+import theme from './RichTooltip.scss';
+
+export function RichTooltipContent(props) {
+	let content;
+
+	if (props.loading) {
+		content = (
+			<div className={theme['tc-center-content']}>
+				<CircularProgress />
+			</div>
+		);
+	}
+
+	if (props.error) {
+		content = (
+			<div className={classNames(theme['tc-center-content'], theme['tc-error-wrapper'])}>
+				<Icon name="talend-warning" className={theme['tc-center-icon']} />
+				<div className={theme['tc-error-body']}>
+					<h1>Whoops !</h1>
+					<div>{props.error}</div>
+				</div>
+			</div>
+		);
+	}
+
+	if (typeof props.Content === 'string' && !(props.error || props.loading)) {
+		content = props.Content;
```

Comment:
Here if i'm not mistaking Content is a string but with an Uppercase, that's not semanticaaly correct for props.